Docker/README.md

```diff
@@ -153,17 +153,16 @@ Note that prebuilt images are less recent and only available for x64 (Intel, AMD
 # First time only
 git clone https://github.com/FreshRSS/FreshRSS.git
 
-cd ./FreshRSS/
+cd FreshRSS/
 git pull
-sudo docker pull ubuntu:18.10
```

Comment:
When building the image locally, pulling the base image makes sure it is up to date.

Reply:
Maybe this variant of the build command ensures we don't need to update the version number at two places and there is only one truth.
````
docker build --help
.....
--pull                    Always attempt to pull a newer version of the image
````

So I would change the following line to:
````
docker build --pull --tag freshrss/freshrss -f Docker/Dockerfile .
````

Is this okay for you?

Reply:
Yes, that would be even better 👍